app/views/group_assignment_invitations/successful_invitation.html.erb

```diff
@@ -14,6 +14,7 @@
 <div class="site-content-body">
   <div class="markdown-body">
     <h2>You are ready to go!</h2>
+    <p>If this is your first assignment, you will recieve an email to join the <%= link_to "@#{decorated_organization.login}", decorated_organization.github_url %>. You can ignore that.</p>
```

Comment:
Same as above.

Can we only show this message in cases where we have added the user to the organization? If not, how about this:

> You may receive an invitation to join `@organization` via email. We have already accepted the invitation on your behalf. No further action is necessary.